### DISCUSSION TITLE
Sistema operacional

### DISCUSSION THREAD

**Discussion by WarZbr** [community/community#142088] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Estou com duvida em como resolver exercicios de memoria(paginação e segmentação) então se alguem puder me explicar como funciona a tabela de pagina final 
[exercicio memoria alocacao de paginas.xlsx](https://github.com/user-attachments/files/17452124/exercicio.memoria.alocacao.de.paginas.xlsx)

**Comment by ebndev** [comment]
Hi @WarZbr, thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md). 

We’ll be locking this post for now, but please feel encouraged to create a new discussion in English if you still need support.
